src/eth.rs

```diff
@@ -25,8 +25,12 @@ pub enum EtherType {
     Loop = 0x0060_u16.to_be(),
     Ipv4 = 0x0800_u16.to_be(),
     Arp = 0x0806_u16.to_be(),
+    VlanTag = 0x8100_u16.to_be(),
     Ipv6 = 0x86DD_u16.to_be(),
+    Ieee8021q = 0x88A8_u16.to_be(),
+    Ieee8021aeMacSec = 0x88E7_u16.to_be(),
     FibreChannel = 0x8906_u16.to_be(),
     Infiniband = 0x8915_u16.to_be(),
     LoopbackIeee8023 = 0x9000_u16.to_be(),
+    Ieee8021qQinQ = 0x9100_u16.to_be(),
```

Comment:
Linux headers specify three types of QinQ:

https://elixir.bootlin.com/linux/v6.14.6/source/include/uapi/linux/if_ether.h#L114

Can we add all of them, or at least, specify this one as `Iee8021QinQ1`?

My suggestion:

```suggestion
    Ieee8021QinQ1 = 0x9100_u16.to_be(),
    Ieee8021QinQ2 = 0x9200_u16.to_be(),
    Ieee8021QinQ3 = 0x9300_u16.to_be(),
```

Reply:
Done.